kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -219,6 +220,15 @@
           }, this.pollingInterval);
         }
       },
+      setPollingInterval(status) {
+        if (status === SyncStatus.QUEUED) {
+          // check more frequently for updates if the user is waiting to sync,
+          // so that the sync isn't missed
+          this.pollingInterval = 1000;
+        } else {
+          this.pollingInterval = 10000;
```

Comment:
Another thing we should maybe do, is prevent polling when the drop down menu is hidden. So perhaps conditionalizing polling on `userMenuDropdownIsOpen`.